modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -219,6 +219,9 @@ public void processOpts() {
             importMapping.remove("DateTime");
             instantiationTypes.put("array", "java.util.ArrayList");
             instantiationTypes.put("map", "java.util.HashMap");
+        } else {
+            languageSpecificPrimitives.add("List");
+            languageSpecificPrimitives.add("Map");
```

Comment:
Hmm, looks like having those as primitives removes the imports. I'll have to figure out how this works ...
Likely I'll remove this primitive-definition here and instead add some special-casing code to `getSwaggerType`.